container/build-oci-image

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+ctr=$(buildah from docker.io/library/ubuntu:rolling)
+buildah add "$ctr" entrypoint.sh /
+buildah config --cmd '["lemonade-server-dev"]' "$ctr"
+buildah config --entrypoint '["/entrypoint.sh"]' "$ctr"
+buildah config --port 8000 "$ctr"
+buildah run "$ctr" /bin/sh -c 'ln -sf /usr/share/zoneinfo/Europe/London /etc/localtime; \
+     apt update; \
+     apt install -y \
+     git \
```

Comment:
This is missing curl and libcurl4-openssl-dev to be able to start the vulkan backend.